version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "partner_tag": "v4.5.5",
-  "claimFormat": "v0.3.0",
-  "parserTag": "v0.3.1"
+  "partner_tag": "v4.5.7",
```

Comment:
this seems wrong, we should keep the latest versions

Reply:
Fixed.